pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -43,6 +43,8 @@ var _ apis.Defaultable = (*PipelineRun)(nil)
 // PipelineRunSpec defines the desired state of PipelineRun
 type PipelineRunSpec struct {
 	PipelineRef PipelineRef `json:"pipelineRef"`
+	// +optional
```

Comment:
Should `PipelineRef` also be marked optional?

Reply:
👍 yup, similar to `TaskRun`:

```	// +optional
	TaskRef *TaskRef `json:"taskRef,omitempty"`
	// +optional
	TaskSpec *TaskSpec `json:"taskSpec,omitempty"`
```